notebooks/en/structured_generation.ipynb

```diff
@@ -0,0 +1,600 @@
+{
```

Comment:
<p>"<a href="https://github.com/outlines-dev/outlines/" target="_blank">Outlines</a> is the library that runs <em>under</em> the hood <em>of</em> our Inference API to constrain output generation."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>